staging.midrc.org/manifest.json

```diff
@@ -168,7 +168,7 @@
     "kube_bucket": "kube-midrcprod-gen3",
     "dispatcher_job_num": "10",
     "logs_bucket": "logs-midrcprod-gen3",
-    "dictionary_url": "http://s3.amazonaws.com/dictionary-artifacts/midrc_dictionary/0.6.6/schema.json",
+    "dictionary_url": "http://s3.amazonaws.com/dictionary-artifacts/midrc_dictionary/0.6.7/schema.json",
```

Comment:
i don't know the midrc release process! but i see this version has not been deployed to qa or staging, is that ok?